networking/using-ptp.adoc

```diff
@@ -76,6 +76,11 @@ include::modules/cnf-configuring-the-ptp-fast-event-publisher.adoc[leveloffset=+
 
 include::modules/cnf-fast-event-notifications-api-refererence.adoc[leveloffset=+2]
 
+[role="_additional-resources"]
+.Additional resources
+
+* For a reference consumer application that uses cloud native events, see the example consumer application in link:https://github.com/redhat-cne/cloud-event-proxy/tree/main/examples/consumer[Sample `cloudNotifications` application in Go].
```

Comment:
Same comment here about needing to get this GitHub link approved before it can be included

Reply:
Understood! Will remove pending ack from Eric.